index.js

```diff
@@ -441,6 +446,9 @@ function windowCreateFinish(json) {
     elements[json.targetID].on('closed', () => {
         client.write(json.targetID, consts.eventNames.windowEventClosed)
         delete elements[json.targetID]
+        setTimeout(() => {
```

Comment:
I don't think this part is consistent with apps that use multiple windows. Why do you need this logic?